packages/components/document-card/src/document-card.js

```diff
@@ -15,7 +18,8 @@ customElementDefineHelper(
 				name: { type: String, reflect: true },
 				description: { type: String, reflect: true },
 				selected: { type: Boolean, reflect: true },
-				mobileDescription: { attribute: 'mobile-description', type: String, reflect: true }
+				mobileDescription: { attribute: 'mobile-description', type: String, reflect: true },
+				cardType: { type: String, reflect: true }
```

Comment:
Why not just using `type`? Since it's a document-CARD it seems a bit redundant.

Reply:
ok